source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+<b>Technical Description</b>
+
+N/A
+
+##G0035
+
+<b>Functional Description</b>
+
+The Holders of the Transit Procedure can only request information on their own Guarantee[s].
+
+<b>Technical Description</b>
+
+N/A
+
+##G0036
+
+<b>Functional Description</b>
+
+COMPETENT CUSTOMS OFFICE AT DEPARTURE which is authorised for Enquiry communicates only with COMPETENT CUSTOMS OFFICE AT DESTINATION which is authorised for Enquiry. COMPETENT CUSTOMS OFFICE AT DEPARTURE which is authorised for Recovery communicates only with COMPETENT CUSTOMS OFFICE AT DESTINATION which is authorised for Recovery.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0038
+
+<b>Functional Description</b>
+
+The Header shall be included if the field &lt;TRANSIT OPERATION.MRN&gt; or &lt;TRANSIT OPERATION.LRN&gt; is PRESENT in the rejected message and shall be included in the CC917C.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0039
+
+<b>Functional Description</b>
+
+If the field &lt;TRANSIT OPERATION.LRN&gt; is PRESENT in the rejected message, this Data Item shall be filled in. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0040
+
+<b>Functional Description</b>
+
+If the field &lt;TRANSIT OPERATION.MRN&gt; is PRESENT in the rejected message, this Data Item shall be filled in. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0041
+
+<b>Functional Description</b>
+
+In a case when National risk is identified, by default, the value ‘X’ shall be included in &lt;RiskAnalysisIdentification.code&gt; when generated by the risk engine. However, if the sender would likely consider such information sent by the Office of Export /Office of Departure as useless for the recipient, then Office of Export /Office of Departure should use the value ‘N’.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0042
+
+<b>Functional Description</b>
+
+This D.G./D.I. is not used to report discrepancies. The Control Message always reports back D.G./D.I as at declaration message. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0045
+
+<b>Functional Description</b>
+
+The information in this Data Group/Data Item will override the information included in the CC015C (or in the latest CC013C, if any).
+
+<b>Technical Description</b>
+
+N/A
+
+##G0050
+
+<b>Functional Description</b>
+
+The Reference number shall include the ARC number or the fallback eAD reference number when the ‘Type’ of the ‘Additional reference’ is C651 or C658 respectively.
+
```

Comment:
Is `eAD` correct? Should it not be `EAD`?

Insert a comma before `respectively`.